back/app/controllers/web_api/v1/projects_controller.rb

```diff
@@ -51,6 +51,25 @@ def index
     )
   end
 
+  def moderatable
```

Comment:
I'm sorry but I got sick of waiting 50 seconds every time I wanted to use the project select. I just created a very quick endpoint to request projects that you are allowed to moderate instead. Let me know if this is a really really bad idea or something